### DISCUSSION TITLE
Update docs code in the website (Not in github)

### DISCUSSION THREAD

**Discussion by yasindunethmina** [TanStack/query#4354] [General] [answered]
I've noticed an error at the first line of the code importing react query in the docs (https://react-query-v3.tanstack.com/overview). It's the wrong way, it should be replaced with tanstack word, this may confuse newcomers.

**Comment by thiswallz** [comment]
agreed, 

correct way is:

`import { useQuery } from '@tanstack/react-query';`

**Comment by TkDodo** [comment, marked as answer]
you're looking at the v3 docs when you probably want the v4 docs:

https://tanstack.com/query/v4/docs/overview
